pages/_en-US/twilightmenu/custom-fonts.md

```diff
@@ -22,6 +22,11 @@ Custom fonts are loaded from `sd:/_nds/TWiLightMenu/extras/fonts/[font name]/[fo
 
 The `-ds` and `-dsi` files have higher priority than the normal one and if found will be used when TWiLight Menu++ is running in DS or DSi Mode respectively.
 
+### Skin fonts
+You may add fonts to [custom DSi/3DS theme skins](custom-dsi-3ds-skins), which will override whatever is set in TWiLight Menu++ settings. These fonts go in the `font` folder, in the skin's root folder.
```

Comment:
I understand the way you wrote the sentence here is more consistent with the way the custom skins page is written, but within the context of this page, it's definitely weird. Perhaps try something of the sort to line your additions with the official page:

```md
### Skin-Dependent fonts
[Custom skins](custom-dsi-3ds-skins) for the Nintendo DSi/Nintendo 3DS themes can override the font setting for the general UI. These overrides are to be placed within the skin's `font` folder.
- Specific elements could have their font overridden, by titling the file according to the specifications set by the page above
- Delete the files within the skin in order to restore the font to whatever was specified in the TWiLight Menu++ settings
```